eth/fetcher/block_fetcher.go

```diff
@@ -63,12 +65,16 @@ var (
 	bodyFilterOutMeter   = metrics.NewRegisteredMeter("eth/fetcher/block/filter/bodies/out", nil)
 )
 
+func GetMeters() (int64, int64) {
```

Comment:
Seems unused function?

Reply:
Right, leftover crap from when I experimented with a testcase, and wanted to gain access to the metrics. I'll remove it